source/gui/guiHelper.py

```diff
@@ -290,19 +290,34 @@ def addLabeledControl(self, labelText, wxCtrlClass, **kwargs):
 			self.addItem(labeledControl.sizer)
 		return labeledControl.control
 
-	def addDialogDismissButtons(self, buttons):
+	def addDialogDismissButtons(self, buttons, separated=False):
 		""" Adds and aligns the buttons for dismissing the dialog; e.g. "ok | cancel". These buttons are expected
 		to be the last items added to the dialog. Buttons that launch an action, do not dismiss the dialog, or are not
 		the last item should be added via L{addItem}
-		@param buttons: the buttons to add
-		@type buttons: wx.Sizer or guiHelper.ButtonHelper or single wx.Button
+		@param buttons: The buttons to add
+		@type buttons:
+		  wx.Sizer or guiHelper.ButtonHelper or single wx.Button
+		  or a bit list of the following flags: wx.OK, wx.CANCEL, wx.YES, wx.NO, wx.APPLY, wx.CLOSE,
+		  wx.HELP, wx.NO_DEFAULT
+		@param separated:
+		  Whether a separator should be added between the dialog content and its footer.
+		  Should be set to L{False} for message or single input dialogs, L{True} otherwise.
+		@type separated: L{bool}
 		"""
+		if self.sizer.GetOrientation() != wx.VERTICAL:
+			raise NotImplementedError(
+				"Adding dialog dismiss buttons to a horizontal BoxSizerHelper is not implemented."
+			)
 		if isinstance(buttons, ButtonHelper):
 			toAdd = buttons.sizer
 		elif isinstance(buttons, (wx.Sizer, wx.Button)):
 			toAdd = buttons
+		elif isinstance(buttons, int):
+			toAdd = self._parent.CreateButtonSizer(buttons)

```

Comment:
As per my other comment about translations of the button labels, perhaps it would be better to construct the buttons with our own translated labels here?